ai-service/src/ai_service/db.py

```
@@ -44,4 +39,37 @@ def add_chunks(chunks: list[str], embeddings: list[list[float]]) -> None:
             ids=ids,
         )
     except Exception as e:
-        raise DatabaseError(f"Failed to add chunks: {e}") from e
+        raise errors.DatabaseError(f"Failed to add chunks: {e}") from e
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)